### PR TITLE
Add documentation for setspeed and reversespeed triggers

### PR DESCRIPTION
**Pull request by 879m**
Adds `trigger_momentum_setspeed` and `trigger_momentum_reversespeed` documentation.

### CODE REVIEW

File: _posts/entities/2021-01-06-trigger_momentum_setspeed.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+>**Update every intervals** (OnThink&lt;**integer**&gt;)
+
+ If the trigger should update every interval.
+
+>**Interval** (Interval&lt;**float**&gt;)
+
+ The length of an interval in seconds.
+
+>**Every tick** (EveryTick)&lt;**integer**&gt;)
```
Comment (braem) [inline _posts/entities/2021-01-06-trigger_momentum_setspeed.md:54]:
```suggestion
>**Every tick** (EveryTick&lt;**integer**&gt;)
```

File: _posts/entities/2021-01-06-trigger_momentum_setspeed.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+tags:
+ - trigger
+---
+
+
+----
+
+
+A [trigger](https://developer.valvesoftware.com/wiki/Triggers){:target="blank"} for setting the speed of the player in a particular axis or direction.
+If you want send the player in a specific direction, use a [catapult](/entity/trigger_momentum_catapult/) instead.
```
Comment (braem) [inline _posts/entities/2021-01-06-trigger_momentum_setspeed.md:13]: Not 100% on this but should this also mention a push trigger?

**Review by braem — CHANGES_REQUESTED** [review]
Looks good but is missing default values for each KV.

File: _posts/entities/2021-01-06-trigger_momentum_setspeed.md (hunk 1 above)
Comment (879m) [reply _posts/entities/2021-01-06-trigger_momentum_setspeed.md]: I was basing it off other doc pages, but they're included now.

File: _posts/entities/2021-01-06-trigger_momentum_setspeed.md (hunk 2 above)
Comment (879m) [reply _posts/entities/2021-01-06-trigger_momentum_setspeed.md:14]: Agree, I've added that.

File: _posts/entities/2021-01-06-trigger_momentum_setspeed.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+tags:
+ - trigger
+---
+
+
+----
+
+
+A [trigger](https://developer.valvesoftware.com/wiki/Triggers){:target="blank"} for setting the speed of the player in a particular axis or direction.
+If you want to change the player's velocity, use a [push trigger](/entity/trigger_momentum_push/) insetad.
```
Comment (braem) [inline _posts/entities/2021-01-06-trigger_momentum_setspeed.md:13]:
```suggestion
If you want to change the player's velocity, use a [push trigger](/entity/trigger_momentum_push/) instead.
```

**Review by braem — CHANGES_REQUESTED** [review]
Just one more :)
Feel free to use the "Commit suggestion" thingie. Commits will be squashed anywho.

File: _posts/entities/2021-01-06-trigger_momentum_setspeed.md (hunk 1 above)
Comment (braem) [reply _posts/entities/2021-01-06-trigger_momentum_setspeed.md]: Yeah I (or someone else :O) needs to go through and standardize it all.

**Review by braem — APPROVED** [review]

**Pull request merged**